packages/atomic/src/components/search/atomic-did-you-mean/atomic-did-you-mean.tsx

```diff
@@ -52,7 +52,11 @@ export class AtomicDidYouMean implements InitializableComponent {
   }
 
   private withQuery(
-    key: 'no-results-for' | 'query-auto-corrected-to' | 'showing-results-for',
+    key:
+      | 'no-results-for'
+      | 'no-results-for-dym'
```

Comment:
I prefer to avoid abbreviations and use the full words.
`no-results-for-did-you-mean`